src/rules/hindi.toml

```diff
@@ -0,0 +1,19 @@
+min_trimmed_length = 0
+min_word_count = 0
```

Comment:
same goes for this one, are there valid sentences with one word?

Reply:
Yes, one word is a valid sentence.